notebooks/notebooks.yaml

```diff
@@ -0,0 +1,14 @@
+stages:
+  results-mind-val:
+    cmd: jupytext --to notebook --execute mind-val.md
+    deps:
+      - notebooks
+    outs:
+      - notebooks/mind-val.ipynb
```

Comment:
Needs to be relative to this directory, not project root.

Also, let's make this output _uncached_ so we can browse the notebooks on GitHub:

```
- mind-val.ipynb:
    cache: false
```